app/models/concerns/outgoings_summarisable.rb

```diff
@@ -32,36 +44,56 @@ def not_all_child_dependants_are_over_16
   end
 
   def dependants_allowance_over_16
+    return if asylum_support? || !client_income_relevant?
+
     # We should show this field if there are any child dependants without income, as we assume all child dependants without income are over 16
-    from_cfe_payload("result_summary.disposable_income.dependant_allowance_over_16") if client_income_relevant? && (adult_dependants || dependant_incomes.present?)
+    adult_dependants || dependant_incomes.present? ? from_cfe_payload("result_summary.disposable_income.dependant_allowance_over_16") : 0
   end
 
   def client_tax_and_national_insurance
-    tax_and_national_insurance if client_income_relevant? && employed?
+    return if asylum_support? || !client_income_relevant?
+
+    employed? ? tax_and_national_insurance : 0
   end
 
   def partner_tax_and_national_insurance
-    tax_and_national_insurance("partner_") if partner_income_relevant? && partner_employed?
+    return if asylum_support? || !client_income_relevant?
+
+    if partner
+      partner_employed? ? tax_and_national_insurance("partner_") : 0
+    end
   end
 
   def client_employment_deduction
-    employment_deduction if client_income_relevant? && employed?
+    return if asylum_support? || !client_income_relevant?
+
+    employed? ? employment_deduction : 0
   end
 
   def partner_employment_deduction
-    employment_deduction("partner_") if partner_income_relevant? && partner_employed?
+    return if asylum_support? || !client_income_relevant?
+
+    if partner
+      partner_employed? ? employment_deduction("partner_") : 0
+    end
   end
 
   def client_maintenance_allowance
-    from_cfe_payload("result_summary.disposable_income.maintenance_allowance") if client_income_relevant?
+    return if asylum_support? || !client_income_relevant?
+
+    from_cfe_payload("result_summary.disposable_income.maintenance_allowance") || 0
   end
 
   def partner_maintenance_allowance
-    from_cfe_payload("result_summary.partner_disposable_income.maintenance_allowance") if partner_income_relevant?
+    return if asylum_support? || !client_income_relevant?
+
+    from_cfe_payload("result_summary.partner_disposable_income.maintenance_allowance") || 0 if partner
```

Comment:
So here we return nil if `asylum_support?` or `!client_income_relevant?` or (implicitly) `!partner`. But we already had a single convenience method that did all of this that we were previously using - `partner_income_relevant?`. So let's use that.